watertap/examples/flowsheets/RO_with_energy_recovery/RO_with_energy_recovery.py

```diff
@@ -82,9 +93,27 @@ def build():
     m.fs.S1 = Separator(
         default={"property_package": m.fs.properties, "outlet_list": ["P1", "PXR"]}
     )
-    m.fs.P1 = Pump(default={"property_package": m.fs.properties})
+
+    if variable_efficiency is VariableEfficiency.none:
+        m.fs.P1 = Pump(default={"property_package": m.fs.properties})
+        m.fs.P2 = Pump(default={"property_package": m.fs.properties})
+    elif variable_efficiency is VariableEfficiency.flow:
+        m.fs.P1 = Pump(
+            default={
+                "property_package": m.fs.properties,
+                "variable_efficiency": VariableEfficiency.flow,
+            }
+        )
+        m.fs.P2 = Pump(
+            default={
+                "property_package": m.fs.properties,
+                "variable_efficiency": VariableEfficiency.flow,
+            }
+        )
+    else:
+        raise NotImplementedError("Variable efficiency type not recognized")
```

Comment:
```suggestion
  
       
      m.fs.P1 = Pump(
          default={
              "property_package": m.fs.properties,
              "variable_efficiency": variable_efficiency,
          }
      )
      m.fs.P2 = Pump(
          default={
              "property_package": m.fs.properties,
              "variable_efficiency": variable_efficiency,
          }
      )
```
I think you can just pass the variable_efficiency arg into the pump config option, and error handling should/could be caught by the pump unit itself.